auslib/blobs/apprelease.py

```diff
@@ -306,6 +306,22 @@ def getApplicationVersion(self, platform, locale):
         may have been a pretty version for users to see"""
         return self.getExtv(platform, locale)
 
+    def getReferencedReleases(self):
+        """
+        :return: Returns set of names of partially referenced releases that the current
+        release references
+        """
+        referencedReleases = set()
+        if self.get('platforms'):
```

Comment:
You can remove a lot of the nesting here by using statements like "for platform in self.get("platforms", {})".